src/python/ensembl/production/datafile/scraper/parsers/bam_parser.py

```diff
@@ -0,0 +1,47 @@
+#  See the NOTICE file distributed with this work for additional information
```

Comment:
I would name the file `bam.py` so the include become `from ensembl.production.datafile.scraper.parser import bam`